test/test_go.rb

```diff
@@ -1935,61 +1903,54 @@ def test_dir_completion
     tmux.send_keys 55
     tmux.until { |lines| lines.match_count == 1 }
     tmux.send_keys :Enter
-    tmux.until(true) { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/' }
     tmux.send_keys :xx
-    tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/xx' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/xx' }
 
     # Should not match regular files (bash-only)
     if self.class == TestBash
       tmux.send_keys :Tab
-      tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/xx' }
+      tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/xx' }
     end
 
     # Fail back to plusdirs
     tmux.send_keys :BSpace, :BSpace, :BSpace
-    tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55' }
     tmux.send_keys :Tab
-    tmux.until { |lines| lines[-1] == 'cd /tmp/fzf-test/d55/' }
+    tmux.until { |lines| lines[-1] == '$ cd /tmp/fzf-test/d55/' }
   ensure
     FileUtils.rm_r '/tmp/fzf-test'
   end
 
   def test_process_completion
-    tmux.send_keys 'sleep 12345 &', :Enter
-    lines = tmux.until { |lines| lines[-1].start_with? '[1]' }
-    pid = lines[-1].split.last
     tmux.prepare
-    tmux.send_keys 'C-L'
+    tmux.send_keys 'sleep 12345 &', :Enter
+    pid = nil
+    tmux.until { |lines| pid = /(?<= )\d+/.match lines[-2] }
     tmux.send_keys 'kill ', :Tab
-    tmux.until { |lines| lines.match_count.positive? }
+    tmux.until { |lines| lines.item_count.positive? }
```

Comment:
Can you explain why you changed `match_count` to `item_count`? Both will work here.

Reply:
Yes, no difference, `item_count` just seemed more "correct" before entering any search terms?